Remove inappropriate "please", minor rewording (#4249)
iterative/dvc.org: content/docs/use-cases/versioning-data-and-models/tutorial.md

> We have last tested this tutorial with Python 3.7.

You'll need [Git](https://git-scm.com/) to run the commands in this tutorial.

> Also, if DVC is not installed, you can follow these [instructions](/doc/install)
> to do so.

Reviewer: ```suggestion
Also, [follow these instructions to install DVC](/doc/install) if it's not
already installed.
```

> > If you're using Windows, you can review
> > [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows) for
> > important tips to improve your experience.

Reviewer: ```suggestion
> See [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows)
> for important tips to improve your experience on Windows.
```
